decidim-conferences/config/locales/en.yml

```
@@ -288,7 +288,23 @@ en:
           error: There was a problem updating a registration type for this conference.
           success: Conference registration type successfully updated.
       titles:
+        conference_admins: Conference admins
+        conference_invites: Conference invites
+        conference_partners: Conference partners
+        conference_registrations: Conference registrations
+        conference_speakers: Conference speakers
         conferences: Conferences
+        duplicate_conference: Duplicate - %{title} - conference
+        media_links: Media links
+        new_conference: New conference
+        new_conference_admin: New conference admin
+        new_conference_invite: New conference invite
```

For consistency with the views' title:

```suggestion
        new_conference_invite: Invite participant
```